api/desecapi/models.py

```diff
@@ -99,6 +99,7 @@ class Domain(models.Model):
     arecord = models.CharField(max_length=255, blank=True)
     aaaarecord = models.CharField(max_length=1024, blank=True)
     owner = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='domains')
+    acme_challenge = models.CharField(max_length=255, blank=True)
```

Comment:
updating any db that was build with the previous version of this PR will fail, drop database then rebuild from scratch (or change manually).